rust-runtime/aws-smithy-runtime-api/src/client/auth.rs

```diff
@@ -187,6 +187,8 @@ impl AuthScheme for SharedAuthScheme {
     }
 }
 
+impl ValidateConfig for SharedAuthScheme {}
```

Comment:
should these be delegating to their inner members? (either now or eventially)

Reply:
Eventually, yes. I figure we'll add them as they're needed.